Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/burger.yml

```diff
@@ -44,3 +46,5 @@
   - type: Sprite
     sprite: _NF/Objects/Consumable/Food/burger.rsi
     state: rosy-mothroach
+  - type: Food
+    quality: Nasty
```

Comment:
Why is the regular mothroach burger high-quality while this is nasty? Shouldn't the rare and exotic version be even better?!